src/SFA.DAS.AANHub.Database/Tables/MemberRegion.sql

```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [dbo].[MemberRegion]
+(
+	[MemberId] UNIQUEIDENTIFIER NOT NULL, 
+    [RegionId] INT NOT NULL
+	PRIMARY KEY (MemberId, RegionId)
```

Comment:
Add primary key name `PK_MemberRegion`

Reply:
![image](https://user-images.githubusercontent.com/28269309/208629895-441c6d06-4403-4a31-bcb6-111046312c19.png)

The formatting seems to just be a Github issue than a formatting issue